src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -302,9 +302,9 @@ const WorkspaceDashboard = _.flow(
     } else {
       loadAzureStorage()
 
-      // sas tokens expires after 1 hour
+      // sas tokens expires after 8 hours
       clearInterval(sasTokenRefreshInterval.current)
-      sasTokenRefreshInterval.current = setInterval(loadAzureStorage, Utils.durationToMillis({ minutes: 50 }))
+      sasTokenRefreshInterval.current = setInterval(loadAzureStorage, Utils.durationToMillis({ hours: 8 }))
```

Comment:
This controls the interval at which we refetch the URL… it has nothing to do with the length the URL is valid.

The length of the SAS token needs to be set in the AJAX call to fetch the token (to override the default of 1 hour). And I think we can remove this automatic refreshing of the URL because I don't think we should expect that someone will stay on the same page for 8 hours. Plus we are now showing a message that directs the user to refresh the page to get a new SAS token.

Reply:
You can see how to specify the duration by looking at the swagger documentation (it will be a URL parameter): https://workspace.dsde-dev.broadinstitute.org/swagger-ui.html#/ControlledAzureResource/createAzureStorageContainerSasToken